app-ios/Sources/CommonComponents/LoadingView.swift

```diff
@@ -0,0 +1,14 @@
+import SwiftUI
+
+public struct LoadingView: View {
```

Comment:
This view can be used in `SponsorView` as well.
Please apply this too 🙏🏼

Reply:
I applied `LoadingView` at https://github.com/DroidKaigi/conference-app-2022/pull/656/commits/c7c73014c48543fb89b651526570ff2432fb1620 !

Reply:
I think  `AssetColors.background.swiftUIColor` should be used here, is it not appropriate?